ex5.2_piece_of_cake.py

```diff
@@ -0,0 +1,11 @@
+#function that calculate the price for all the integers
+def get_recipe_price(prices=dict, optionals=list(), **quantity):
+    sum_price = 0
+    for key in prices.keys():
```

Comment:
you can iterate over keys and values of a dictionary at once 
![image](https://user-images.githubusercontent.com/13467749/159540842-3e7c5c2f-03e3-4727-8644-94a95c4b2189.png)